lib/datadog/appsec/processor/context.rb

```diff
@@ -66,6 +67,19 @@ def finalize
 
         private
 
+        def report_execution(result)
+          Datadog.logger.debug { "libddwaf execution timed out: #{result.inspect}" } if result.timeout
+
+          if LIBDDWAF_SUCCESSFUL_EXECUTION_CODES.include?(result.status)
+            Datadog.logger.debug { "libddwaf execution result: #{result.inspect}" }
+          else
+            message = "libddwaf execution error: #{result.status.inspect} with #{result.inspect}"
```

Comment:
After discussion with @TonyCTHsu we agreed to change it into the following:

```
message = "libddwaf:1.14.0 execution error: #{result.status.inspect}"
```

we will drop `result.inspect` to reduce cardinality and also it doesn't bring much value.